internal/renderer/helm.go

```diff
@@ -241,17 +269,38 @@ func makeStack(appname string, targetDir string, data []byte) error {
 		return errors.Wrap(err, "failed to load template compose")
 	}
 	os.Mkdir(filepath.Join(targetDir, "templates"), 0755)
-	stackSpec := templateconversion.FromComposeConfig(rendered)
-	stack := templatev1beta2.Stack{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "stacks.compose.docker.com",
-			APIVersion: "v1beta2",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      internal.AppNameFromDir(appname),
-			Namespace: "default", // FIXME
-		},
-		Spec: stackSpec,
+	var stack interface{}
+	if !beta1 {
```

Comment:
Same here — and it looks like it's duplicated from above, we should extract it

Reply:
Actually, this instantiates a `templatev1beta2.Stack` whereas the code in `helmRender` above instantiates a `v1beta2.Stack`

Reply:
arf right :sweat: